src/components/Layout.tsx

```diff
@@ -14,28 +14,26 @@ type Props = {
 const Layout = ({ text, icon, children }: Props) => {
   return (
     <>
-      <Flex alignItems="center" h={36} bg="teal.500">
+      <Flex bgColor="teal.500" h={36} align="center">
```

Comment:
ヘッダー部を上下センタリングしておく必要性を感じないから、外側の `Flex` の配置指定(子)は デフォルトの `start` のままで、内側の `Flex` の配置指定(自身) を `center` でもいい気がする

```jsx
<Flex align="center">
  <Flex />
  <Box alignSelf="flex-start" />
</Flex>
```
↓↓↓
```jsx
<Flex>
  <Flex alignSelf="center" />
  <Box />
</Flex>
```

```suggestion
      <Flex bgColor="teal.500" h={36}>
```